src/components/Checkout/Checkout.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <h1>My Cart</h1>
+    <h3>You signed up for the following events:</h3>
+    <div class="component-container">
+      <div class="events component-wrapper">
+        <events />
+      </div>
+      <div class="payment component-wrapper"></div>
+      <div class="codes component-wrapper"></div>
+    </div>
+    <the-footer />
```

Comment:
the-footer should be moved to app.vue if its not already there.

Reply:
gotcha

Reply:
yep, will do